apps/finance/models.py

```diff
@@ -9,22 +9,42 @@
 
 class Transaction(base_models.BaseModel):
     class Transaction_Status(models.TextChoices):
-        PENDING = "PE", _("Pending"),
-        PAID = "PA", _("Paid"),
-        REFUNDED = "RE", _("Refunded"),
-        FAILED = "FA", _("Failed"),
+        PENDING = (
+            "PE",
+            _("Pending"),
+        )
+        PAID = (
+            "PA",
+            _("Paid"),
+        )
+        REFUNDED = (
+            "RE",
+            _("Refunded"),
+        )
+        FAILED = (
+            "FA",
+            _("Failed"),
+        )
         CANCELLED = "CA", _("Cancelled")
 
     class Payment_Method(models.TextChoices):
-        CASH = "CA", _("Cash"),
-        MOMO = "MO", _("Momo"),
-        CARD = "CD", _("Card"),
+        CASH = (
+            "CA",
+            _("Cash"),
+        )
+        MOMO = (
+            "MO",
+            _("Momo"),
+        )
+        CARD = (
+            "CD",
+            _("Card"),
+        )
         ST_POINTS = "SP", _("St_points")
         REFERRAL_POINTS = "RP", _("Referral_points")
 
-    user = models.ForeignKey(
-        usermodels.User, on_delete=models.CASCADE, related_name="transactions"
-    )
```

Comment:
Any reason why you removed the related name?

Reply:
> Any reason why you removed the related name?

lol, when I was troubleshooting my db. I will add it back